### PR TITLE
Support webui tests

### PR DESCRIPTION
**Pull request by tomaskrizek**
- modifications to template to include necessary packages
- added provisioning job to do proper setup for webui tests
- defined new job to use for running webui tests

The template modifications and provisioning should work fine. I'm still verifying the job definition. Once I verify it and have a passing job, I'll update the PR with a comment.

### CODE REVIEW

**Comment by tomaskrizek** [comment]
@pvomacka Ready for review, successful run: https://github.com/tomaskrizek/freeipa/pull/47

File: ansible/roles/machine/provision_webui/templates/ui_test.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ipa_realm: IPA.TEST
+{% endif %}
+{% endfor %}
+
+# Uncomment when IPA is installed without CA
+#no_ca: True
+
+# Uncomment when IPA is installed without DNS server
+#no_dns: True
+
```
Comment (pvomacka) [inline ansible/roles/machine/provision_webui/templates/ui_test.conf.j2]: Could you please add configuration option for Geckodriver.log ? 

```
# Geckodriver setup:
# =================
# log file has to be somewhere, where user has rights to write into file
geckodriver_log_path: /home/me/.ipa/geckodriver.log
```
and set it as PR-CI needs. We might need to collect this log in the future (or is it possible collect it right now?)

Comment (tomaskrizek) [reply ansible/roles/machine/provision_webui/templates/ui_test.conf.j2]: Why, yes -- we can!

very log

**Comment by tomaskrizek** [comment]
I also added support for xmlrpc Python tests, since they require a similar setup. It's possible to execute them now. This PR is ready for review.

xmlrpc test run in Py2: https://fedorapeople.org/groups/freeipa/prci/jobs/4f3eba70-e19a-11e7-a866-001e4fb5bf1d/report.html

**Comment by pvomacka** [comment]
ACK
thumbs up

**Pull request merged**